cli/tests/test_revset_output.rs

```diff
@@ -138,7 +138,7 @@ fn test_bad_function_call() {
     1 | file(a, "../out")
       |         ^------^
       |
-      = Invalid file pattern: Path "../out" is not in the repo
+      = Invalid file pattern: Path not in the repo
```

Comment:
Before, we also listed the path explicitly as `../out`; should we preserve that here too?

Reply:
I removed it as it's obvious that `"../out"` has something wrong.